pkgs/applications/version-management/git-and-tools/git-fame/default.nix

```diff
@@ -0,0 +1,19 @@
+{ lib, bundlerEnv, ruby, fetchFromGitHub }:
+
+bundlerEnv rec {
+  name = "git-fame-${version}";
+
+  version = "2017-01-14";
+  inherit ruby;
+  gemdir = ./.;
+
+  pname = "git_fame";
```

Comment:
it's possible that setting both the name and pname doesn't interact very well. If the ruby gem is called "git_fame" then it's better to name the package the same way.